src/System.Private.Xml/src/System/Xml/XPath/Internal/XPathArrayIterator.cs

```diff
@@ -84,7 +84,5 @@ public override IEnumerator GetEnumerator()
         {
             return list.GetEnumerator();
         }
-
-        private object debuggerDisplayProxy { get { return index < 1 ? null : (object)new XPathNavigator.DebuggerDisplayProxy(Current); } }
```

Comment:
The purpose of the PR was for a review rather than a merge as I don't have the background to determine whether something is truly unused. i.e. due to regressions, partially implemented code, used only in the debugger, etc.
 
So I expected someone with the appropriate competence  to fork my branch, review and revert as appropriate then merge. Does that sound sensible?